packages/app-builder/src/locales/en/decisions.json

```
@@ -59,5 +59,11 @@
   "rules.description": "Description",
   "rules.error.not_found": "The associated rule info could not be found. Please try again or contact your admin.",
   "rules.show_contextual_values": "Show contextual values",
-  "sanction_check.title": "Sanction check"
+  "sanction_check.title": "Sanction check",
+  "outcome.tag.approved.label": "Approved",
+  "outcome.tag.declined.label": "Declined",
+  "outcome.tag.blocked.label": "Blocked and review",
```

```suggestion
  "outcome.tag.approved.label": "Approve",
  "outcome.tag.declined.label": "Decline",
  "outcome.tag.blocked.label": "Block and review",
```
still have a feeling that this is better